samples/ask-akka-agent/src/main/java/akka/ask/indexer/application/RagIndexingWorkflow.java

```diff
@@ -145,8 +145,14 @@ public Effect<Done> abort() {
   // tag::def[]
   @Override
   public WorkflowDef<State> definition() {
+    return workflow()
+        .addStep(processingFileStep())
+        // the processing step is long-running, so we need to set a big timeout
+        .defaultStepTimeout(Duration.of(20, MINUTES));
```

Comment:
Who tested this last time? Was it @efgpinto ?

I thought it processed one file per step call? How can one file take such long time? Then it transition to the same step, but that should count as a new timeout. Do we have a bug?

Reply:
but you saw it time out? perhaps that was with the default step timeout? I'll change this to 1 minute to make it somewhat more reasonable

Reply:
I saw it timeout yeah, but it was with the default of 5 seconds I think.. maybe I then wrongly assumed things. I can give it another try later after the changes. But I think you're probably right. 1 min should be fine.